src/librustc_codegen_llvm/back/link.rs

```diff
@@ -885,9 +886,9 @@ fn exec_linker(sess: &Session, cmd: &mut Command, out_filename: &Path, tmpdir: &
     }
     let file = tmpdir.join("linker-arguments");
     let bytes = if sess.target.target.options.is_like_msvc {
-        let mut out = vec![];
+        let mut out = Vec::with_capacity(1 + args.len());
```

Comment:
It looks like this will hold the bytes of UTF-16; should it then be `2*args.len`?

Reply:
Actually it should be `(1 + args.len()) * 2`, as each step of the `for` loop pushes 2 `u8`s. Thanks, I missed that.